hack/e2e.sh

```diff
@@ -408,6 +408,9 @@ function e2e_up() {
   # Create the workload cluster
   create_cluster $WORKLOAD_CLUSTER
 
+  # deploy addons for workload cluster
+  kubectl --kubeconfig ${WORKLOAD_CLUSTER_KUBECONFIG} apply -f manifests/contour/
```

Comment:
I'm wondering if we we should add this change as a follow-up to #28 instead.

Reply:
This change caused the e2e tests to fail, and the change to the production code caused the tests to pass. Per our conversation, we'll keep this change here.